### PR TITLE
CI/GitHub: Use issue form and add contact links

### PR DESCRIPTION
**Pull request by jackpoz**
**Changes proposed:**

-  CI/GitHub: Use issue form and add contact links
-  See https://github.blog/changelog/2021-06-23-issues-forms-beta-for-public-repositories/

This is how https://github.com/TrinityCore/TrinityCore/issues/new will look:
image

This is how "Generic issue" choice will look (mind the small screenshot):
image

**Target branch(es):** 3.3.5/master

- [ ] 3.3.5
- [x] master

**Issues addressed:**

Forces users to fill all the required fields

**Tests performed:**

- Tested the template in another repo
- Tested the github actions that set the labels
- Tested the contact links

**Known issues and TODO list:** (add/remove lines as needed)

- In the future we will modify the links and the templates to fit our needs

### CODE REVIEW

**Comment by jackpoz** [comment]
It can be tested at https://github.com/jackpoz/TrinityCore/issues/new/choose until it gets merged

**Comment by Faq** [comment]
If merged it will work also for 335. like demo shows it?

**Comment by Shauren** [comment]
@Faq yes, this is set on per-repository level, not per-branch

File: .github/ISSUE_TEMPLATE/issue.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      required: true
+  - type: textarea
+    id: commit
+    attributes:
+      label: TC rev. hash/commit
+      description: |
+        Copy the result of server debug command (if you need to run it from client get prat addon)
+    validations:
+      required: true
+  - type: textarea
```
Comment (Carbenium) [inline .github/ISSUE_TEMPLATE/issue.yml:52]: Would a single-line `input` field be enough? Just to reduce the vertical extent a bit.

Comment (jackpoz) [reply .github/ISSUE_TEMPLATE/issue.yml:52]: That could be an idea.

The issue template actually suggests to "Copy the result of server debug command (if you need to run it from client get prat addon)", this is the result of that command:
>TrinityCore rev. c8ca48823d45 2021-06-26 14:21:18 +0200 (3.3.5 branch) (Unix, RelWithDebInfo, Static)
Using SSL version: OpenSSL 1.1.1j  16 Feb 2021 (library: OpenSSL 1.1.1j  16 Feb 2021)
Using Boost version: 1.74.0
Using MySQL version: 80025
Using CMake version: 3.18.4
Compiled on: Linux 5.11.0-1006-azure
Automatic database updates are enabled for the following databases: Auth, Characters, World
Worldserver listening connections on port 8085
Realmlist (Realm Id: 1) configured in port 8085
VMAPs status: Enabled. LineOfSight: 1, getHeight: 1, indoorCheck: 1
MMAPs status: Enabled
maps directory located in /trinity/wowdata/maps. Total size: 291014951 bytes
vmaps directory located in /trinity/wowdata/vmaps. Total size: 657797442 bytes

If we want to allow pasting this, then we need a multi-row field.

Comment (Carbenium) [reply .github/ISSUE_TEMPLATE/issue.yml:52]: I suppose there was a misunderstanding. My comment was meant for the `os` field.
I agree that we need multiline for the `server debug` output.

File: .github/ISSUE_TEMPLATE/issue.yml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        - 3.3.5
+        - master
+    validations:
+      required: true
+  - type: textarea
+    id: commit
+    attributes:
+      label: TC rev. hash/commit
+      description: |
+        Copy the result of server debug command (if you need to run it from client get prat addon)
```
Comment (Faq) [inline .github/ISSUE_TEMPLATE/issue.yml:49]:
```suggestion
        Copy the output of server console (when used ".server debug" command) or from game client (by using "Prat" addon)
```

File: .github/ISSUE_TEMPLATE/issue.yml (hunk 1 above)
Comment (jackpoz) [reply .github/ISSUE_TEMPLATE/issue.yml:52]: @Carbenium I picked the field above instead of the field below. OS single line sounds good

Comment (jackpoz) [inline .github/ISSUE_TEMPLATE/issue.yml:52]:
```suggestion
  - type: input
```

File: .github/ISSUE_TEMPLATE/issue.yml (hunk 2 above)
Comment (jackpoz) [reply .github/ISSUE_TEMPLATE/issue.yml:49]: should `when used ".server debug" command` be `when using ".server debug" command` ?

Comment (Faq) [reply .github/ISSUE_TEMPLATE/issue.yml:49]: using sounds better :)

Comment (offl) [reply .github/ISSUE_TEMPLATE/issue.yml:49]: Suggested version is a bit confusing

Comment (jackpoz) [reply .github/ISSUE_TEMPLATE/issue.yml:49]: As the PR has been merged, feel free to open a new PR changing the text. Both versions sounds a bit confusing to me, maybe we will come up with something that is even clearer :)

**Pull request merged**